src/basics.md

```diff
@@ -1265,6 +1266,32 @@ fn main() {
     println!("Time elapsed in expensive_function() is: {:?}", duration);
 }
 ```
+[ex-convert-datetime]: #ex-convert-datetime
+<a name="ex-convert-datetime"></a>
+## Converting a local time to UTC time and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the local time and displays it using [`Local::now`] and then converts it to the UTC format using the [`DateTime`] struct. A timezone is then constructed using the [`FixedOffset`] struct and the UTC time is the converted to UTC+8 and UTC-8.
+
+```rust
+extern crate chrono;
+
+use chrono::{DateTime, FixedOffset, Local, Utc};
+
+fn main() {
+    let local_time: DateTime<Local> = Local::now();
+    let utc_time: DateTime<Utc> = DateTime::<Utc>::from_utc(local_time.naive_utc(), Utc);
+    let china_timezone: FixedOffset = FixedOffset::east(28_800);
+    let la_timezone: FixedOffset = FixedOffset::west(28_800);
```

Comment:
I'd suggest to add a little more variety to the example by changint the offset in either "east" or "west" wariant